source/includes/_intro.md

```diff
@@ -52,6 +52,7 @@ Error Code | Reason | Description
 403 | Forbidden | Your request is trying to access data it has no credentials for
 404 | Not Found | The specified resource could not be found.
 409 | Conflict | There was a conflict with processing your request. Try later or change the data being submitted
+415 | Unsupported Media Type | Your request's Content-Type isn't conforming.
```

Comment:
What with `The requests' payload is in a format that is not supported by this method on the target resource.` ?

Reply:
I don't know, your phrase means to me something else

hmm :thinking:  for example my data looks like `"{"key1": "value1", "key2": "value2"}"` instead off: 
```
{
  "key1": "value1", 
  "key2": "value2"
}
```

Reply:
The headers are a part of the payload